bluesky/instrument/callbacks/spec_data_file_writer.py

```diff
@@ -48,6 +48,32 @@ def spec_comment(comment, doc=None):
     APS_fw.spec_comment(comment, doc, specwriter)
 
 
+def RE_finder():
+    """
+    Find RunEngine object from namespaces of the stack.
+
+    * Search backwards through each frame in the stack.
+        * Find any RunEngine objects.
+            * Prefer one with name 'RE'.
+            * Otherwise, pick the first one.
+    * Otherwise, return 'None'.
+    
+    suggestion: Hoist to apstools.utils
+    """
+    import inspect
+    from bluesky import RunEngine
```

Comment:
Same as in previous PR. Let's try to make these dependencies global

Reply:
I prefer to encapsulate both these imports to this one function.  If other functions eventually need either, then it is reasonable to move the import to module scope.  It is not necessary at this time to move them to module level.

Reply:
sounds good